test/test_fx.py

```diff
@@ -732,6 +732,33 @@ def forward(self, x):
         self.assertIn('operator.add', graph_str)
         self.assertIn('torch.add', graph_str)
 
+    def test_pretty_print_node(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.param: torch.nn.Parameter = torch.nn.Parameter(
+                    torch.rand(3, 4))
+                self.linear = torch.nn.Linear(4, 5)
+
+            def forward(self, x: torch.Tensor, y: int = 2):
+                return self.linear(x[y] + self.param).clamp(min=0.0, max=1.0)
+
+        traced = symbolic_trace(M())
+
+        printout: List[str] = """%x : torch.Tensor [#users=1] = placeholder[target=x]
+%y : int [#users=1] = placeholder[target=y](default=2)
+%getitem : [#users=1] = call_function[target=operator.getitem](args = (%x, %y), kwargs = {})
+%param : [#users=1] = self.param
+%add_1 : [#users=1] = call_function[target=operator.add](args = (%getitem, %param), kwargs = {})
+%linear_1 : [#users=1] = call_module[target=linear](args = (%add_1,), kwargs = {})
+%clamp_1 : [#users=1] = call_method[target=clamp](args = (%linear_1,), kwargs = {min: 0.0, max: 1.0})
+return clamp_1
+""".splitlines()
```

Comment:
I'm not a fan of long expected strings like this. There's a huge surface of changes outside of the string printout implementation that could jitter this (how do we emit targets? how do we create node names? what types are recorded/how do they represented?) and can cause the test to be flaky. I'd instead check for the presence of stable keywords. For example, for the placeholder nodes we could check for the presence of the strings ('placeholder', 'x') and ('placeholder', 'y'), respectively

We've gone through the cycle of "long expected strings" -> "check subsets of strings" before and one thing that came out of that is [FileCheck](https://github.com/pytorch/pytorch/blob/3fec1e5025d12faa3f7eec91c82f92cfc77c9bee/torch/testing/__init__.py#L11). You can check `test_jit` for example usages of it. It encapsulates checking for various string fragments in order